### ISSUE TITLE
No 424B4 filings found for CIK 0001105533

### ISSUE THREAD

**Issue by shymoon2** [jadchaar/sec-edgar-downloader#67]
jadchaar, Thanks for the great program. 

I find an issue when downloading a specific filing from EDGAR and would like to ask why it doesn't work for this type of filing. 
dl.get("424B4", "0001105533") returns zero and do not download any 424B4 filing. However, there is indeed one exsiting 424B4 filing for this company on EDGAR. 
https://www.sec.gov/edgar/browse/?CIK=1105533&owner=exclude
https://www.sec.gov/Archives/edgar/data/0001105533/000119312514040941/d621316d424b4.htm

For other filing types, I find it works well. How can this issue be addressed?

**Comment by jadchaar** [comment]
Hi @shymoon2 thanks for reporting this. The new SEC Edgar Search API that is at the backbone of this package is quite new and it seems like the SEC is still trying to nail down issues with it. It seems like the full text search tool does not pick up the 424B4: https://www.sec.gov/edgar/search/#/dateRange=all&category=custom&ciks=0001105533&entityName=ARGOS%2520THERAPEUTICS%2520INC%2520(ARGS)%2520(CIK%25200001105533)&forms=424B4, but like you said, the classic EDGAR search tool picks it up.

I also see that the SEC has updated their classic EDGAR tool https://www.sec.gov/edgar/ with an updated version which seems to have some new JSON information that could be of use to this package. I will try and investigate whether this new data provides more accurate information than the current search API I am using from the SEC.

So tl;dr I am going to report this bug to the SEC full text search team and hopefully they can resolve it on their end!

**Comment by shymoon2** [comment]
Thank you so much for your clarification on the issue and reporting the bug in the new Edgar Search API to the SEC. Greatly appreciated!

**Comment by jadchaar** [comment]
Thank you for bringing this up! Working with the SEC Edgar system is an absolute pain since it is stuck in the dark ages, but I will try my best to look into an alternative for the full text search API since it seems to be pretty flaky since I switched this package to use it. Too bad there is so little documentation on the system and a lot of my time is spent reverse engineering how they download filings. So it might take some time for me to get around to this :).

**Comment by jadchaar** [comment]
Hi @shymoon2, the SEC got back to me today and said they have resolved this bug on their end! Obtaining 424B4 filings for CIK 0001105533 now works as expected:

```python
env ❯ python3
Python 3.9.2 (default, Mar 26 2021, 23:22:38)
[Clang 12.0.0 (clang-1200.0.32.29)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from sec_edgar_downloader import Downloader
>>> dl = Downloader()
>>> dl.get("424B4", "0001105533")
1
```

**Comment by shymoon2** [comment]
> Hi @shymoon2, the SEC got back to me today and said they have resolved this bug on their end! Obtaining 424B4 filings for CIK [0001105533](tel:0001105533) now works as expected:
> 
> ```python
> env ❯ python3
> Python 3.9.2 (default, Mar 26 2021, 23:22:38)
> [Clang 12.0.0 (clang-1200.0.32.29)] on darwin
> Type "help", "copyright", "credits" or "license" for more information.
> >>> from sec_edgar_downloader import Downloader
> >>> dl = Downloader()
> >>> dl.get("424B4", "0001105533")
> 1
> ```

Thank you so much for the update. It works well!!
